app/components/avo/panel_component.html.erb

```diff
@@ -9,6 +9,11 @@
         <% end %>
         <div class="text-2xl tracking-normal font-semibold text-gray-800 truncate items-center flex flex-1" data-target="title">
           <span><%= @name %></span>
+          <% if @reload_button %>
+            <%= button_tag data: { controller: "panel-refresh", action: "click->panel-refresh#refresh" }, id: "reloadButton" do %>
+              <%= svg "arrow-path-rounded-square", class: "ml-2 text-2xl h-6", alt: 'Reload' %>
```

Comment:
After the rebase this changes into this.

```suggestion
              <%= svg "heroicons/outline/arrow-path-rounded-square", class: "ml-2 text-2xl h-6", alt: 'Reload' %>
```

Reply:
Ok, I will update it.